domain/machine/types.go

```diff
@@ -46,6 +46,7 @@ type InstanceStatusType int
 
 const (
 	InstanceStatusUnset InstanceStatusType = iota
+	InstanceStatusPending
```

Comment:
suggestion: add a test to ensure this list stays in sync with the `machine_cloud_instance_status_value` table.